### DISCUSSION TITLE
Do not filter new unknown ids if all IDs were previously filtered

### DISCUSSION THREAD

**Discussion by gamebeast2k** [collin80/SavvyCAN#794] [Ideas]
Hello,
is it somehow possible to set or add it so that new frames are not in the filter?

Example:
You pick up the idle activity of the CAN bus, filter everything and then press a button that generates a previously unknown frame and only see this. This would make it much easier to find triggered events, e.g. window down, open Backdoor 

Thx

**Comment by Pelland-Coupe** [comment]
Hi, Not sure if this addresses your question but here's what I found as a way to "capture" new IDs.
In the sniffer window I select "Never expire IDs" then let it run for a while.
I then select "None" in the filters.

After that if a new ID appears the ID is added to the filter window and has a tick against it.
Note that the frame does not display for some reason. However if I deslect the ID and reselect it I can then see the sniff of the frame, assuming it's still running. Otherwise I look for the ID in the main capture.
Regards.
